tests.h

```diff
@@ -26,7 +26,7 @@ struct TestResult {
   t(&failed); \
   if (failed == 0) { \
     tr->passed++; \
-    printf("PASS " #t "\n"); \
+    /*printf("PASS " #t "\n");*/ \
```

Comment:
I like having these lines printed so I know that the tests I've written are actually running :)

Reply:
okey dokey.  I'll keep the change locally otherwise I cant tell what failed, and you can spank me the first time I screw up :-)